src/Evolution/Executables/CurvedScalarWave/EvolveCurvedScalarWave.hpp

```diff
@@ -0,0 +1,355 @@
+// Distributed under the MIT License.
```

Comment:
would it be possible to combine the Metavariables of the CurvedScalarWave with the ScalarWave Metavariables? They seem extremely similar except the interface initialization.

Reply:
Instead of merging them, we can actually just remove the `ScalarWave` system completely and just use the `CurvedScalarWave` system through and through with a `Minkowski` background. The reason we keep the `ScalarWave` system at all is to treat it as an illustrative / educational tool. This was discussed this on the spectre telecon some time ago, and we decided to leave the `CurvedScalarWave` system independent of the `ScalarWave` executable, and neither merge the two nor get rid of the flat space system for the time being